planet/clients/orders.py

```diff
@@ -271,18 +274,72 @@ async def download_asset(self,
                              progress_bar=progress_bar)
         return dl_path
 
+    @staticmethod  # should this be an async function?
```

Comment:
@mkshah605 really good question! Async doesn't usually help as much with local file reads as it does with network requests (the benefits depend on the system's hardware and OS). And we can only hash one string of bytes at a time asynchronously, so no speed up can be had there. To get a real speed up we would need to read files and hash them in different threads, and there we would have to watch out for memory usage.

Reply:
Makes sense. Thanks Sean!!